src/pages/Checkout/CheckoutPage.vue

```diff
@@ -730,7 +771,18 @@ export default {
 					clearTimeout(toastTimeout);
 				}, 1000);
 			}
-		}
+		},
+		getPromoInformationFromBasket() {
+			getPromoFromBasket(this.derivedPromoFund?.id, this.apollo).then(({ data }) => {
+				this.promoData = data?.shop?.promoCampaign;
+			});
+		},
+		verificationComplete() {
+			this.verificationSubmitted = true;
+		},
+		handleVerificationOptOut() {
+			this.showVerification = false;
```

Comment:
This method will also need to set `this.showVerifyRemovePromoCredit = true` which likely means we'll need to add the `verify-remove-promo-credit` component here in a similar fashion as the CCLandingPage does. We can make another ticket for this if you want but both have to be in place before this code is released in prod.

Note: the VerifyRemovePromoCredit component does function on this page already but it's nested down in the OrderTotals component. I don't "think" there will be an issue with including it in both places as we do this on CCLandingPage > InContextCheckout too.